src/styles/header.scss

```diff
@@ -0,0 +1,73 @@
+@import "variables";
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  height: 60px;
+  padding: 0 50px;
+  margin-bottom: 50px;
+
+  font-size: 12px;
+  font-weight: 500;
+
+  box-shadow: 0 2px 4px rgba(0,0,0, 0.05);
+}
+
+.logo {
+  height: 40px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    justify-content: space-around;
+    margin: 0;
+    padding: 0;
+    height: 60px;
+
+    text-transform: uppercase;
+  }
+
+  &__item {
+    margin-left: 20px;
+    list-style: none;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    position: relative;
+
+    display: block;
+    padding: 23px 0;
+
+    line-height: 14px;
+    text-decoration: none;
+    color: black;
+
+    &:hover {
+      color: $linkColor;
+    }
+  }
+
+  .is-active {
```

Comment:
leave this class for tests, but add additional BEM class for styles

Reply:
created modifier for nav__link element (nav__link--active)